core/src/main/java/com/sequenceiq/cloudbreak/service/stack/DefaultStackService.java

```diff
@@ -154,6 +154,12 @@ public Stack create(CbUser user, Stack stack) {
         } else {
             try {
                 savedStack = stackRepository.save(stack);
+                subnetRepository.save(stack.getAllowedSubnets());
+                Set<InstanceGroup> instanceGroups = stack.getInstanceGroups();
+                for (InstanceGroup group : instanceGroups) {
+                    group.setStack(savedStack);
+                }
+                instanceGroupRepository.save(instanceGroups);
```

Comment:
Now that you mention it's not needed at all since the converter already sets the stack, so I'm going to remove this part now.